mainapp/models.py

```diff
@@ -140,7 +141,7 @@ class NGO(models.Model):
     organisation_type = models.CharField(max_length=250, verbose_name="Type of Organization")
     organisation_address = models.TextField(default='', verbose_name="Address of Organization")
     name = models.CharField(max_length=100, verbose_name="Contact Person")
-    phone = models.CharField(max_length=10)
+    phone = PhoneNumberField(verbose_name="Phone (eg +911234567890)")
```

Comment:
`max_length` value is higher for PhoneNumberField. Can you confirm that this works without migration?

Reply:
Yes, @ssat335 I also think that this change would need a migration file.

Reply:
Migration required!!.

Will update with completing all fields to NUMBERS (except person model).